Modules/_tkinter.c

```diff
@@ -65,6 +65,12 @@ Copyright (C) 1994 Steen Lumholt.
 #endif
 #include <tclTomMath.h>
 
+#if defined(TCL_WITH_EXTERNAL_TOMMATH) || (TCL_MAJOR_VERSION >= 9)
```

Comment:
The new API is available since 8.6.10. Would not be better to use it as soon as it is available? Potentially it can support larger integers.
```suggestion
#if defined(TCL_WITH_EXTERNAL_TOMMATH) || (TK_HEX_VERSION >= 0x0806020a)
```

Reply:
I am not familiar with libtommath APIs nor aware what the exact limits are, but I would think that Tcl is more likely to be the limiting factor anyway, as even the deprecated libtommath functions can handle numbers exceeding what Tcl < 9 can usefully convert to and from (due to various 2**31-1 size limitations on strings, etc.).

Given how Tkinter requires the Tcl version but not the patchlevel to match at compile time and runtime (https://github.com/python/cpython/issues/104399#issuecomment-1572956795), it should be fine to enable the new functions on Tcl 8.7; this would also allow not having to check for `TCL_WITH_EXTERNAL_TOMMATH` in this case (although it still has to be checked for #103842).

Reply:
Agree.